src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,163 @@
+package duke;
+
+import task.Task;
+
+import java.util.Dictionary;
+import java.util.Scanner;
+
+/**
+ * This class handles all the printing of the User Interface.
+ * @author Bryan Ong
+ */
+public class Ui {
+
+    private static final String DIVIDER = "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\n";
+
+    private static Scanner sc = new Scanner(System.in);
+
+    /**
+     * This method reads the command entered.
+     * @return String This returns the command line
+     */
+    public static String readCommand() {
+        return sc.nextLine();
+    }
+
+    /**
```

Comment:
Would it be more succinct to shrink below print functions into one and differentiate the print messages using if-else?